R/pkg/R/SQLContext.R

```diff
@@ -149,6 +149,26 @@ createDataFrame <- function(sqlContext, data, schema = NULL, samplingRatio = 1.0
   dataFrame(sdf)
 }
 
+#' Create a DataFrame
```

Comment:
Don't need to duplicate description for createDataFrame and as.DataFrame? "@alias" may help.